DEPLOYMENT_TYPES.md

```diff
@@ -15,9 +16,12 @@ The table below outlines the defaults per `deployment_type`.
 | **Image Streams**                                               | centos                                   | rhel + xpaas                           | N/A                              | rhel                             |
 
 
-**NOTE** `enterprise` deployment type is used for OpenShift Enterprise version
-3.0.x OpenShift Enterprise deployments utilizing version 3.1 and later will
-make use of the new `openshift-enterprise` deployment type.  Additional work to
-migrate between the two will be forthcoming.
+**NOTES**:
+
+- `enterprise` deployment type is used for OpenShift Enterprise version
+3.0.x.
+- OpenShift Enterprise deployments from version 3.1 to 3.?? use the `openshift-enterprise` deployment type.
```

Comment:
OpenShift Enterprise deployments (version 3.1 and 3.2) and OpenShift Container Platform deployments (version 3.3+) use the `openshift-enterprise` deployment type.